qutip/animation.py

```diff
@@ -51,3 +55,517 @@ def anim_wigner_sphere(wigners, reflections=False, *, cmap=None,
+        The figure canvas on which the plot will be drawn.
+
+    ax : a matplotlib axis instance, optional
+        The axis context in which the plot will be drawn.
+
+    Returns
+    -------
+    fig, ani : tuple
+        A tuple of the matplotlib figure and the animation instance
+        used to produce the figure.
+    """
+
+    fig, ani = plot_spin_distribution(P, THETA, PHI, projection, cmap=cmap,
+                                      colorbar=colorbar, fig=fig, ax=ax)
+
+    plt.close(fig)
+
+    return fig, ani
+
+
+def anim_qubism(ket, theme='light', how='pairs', grid_iteration=1,
+                legend_iteration=0, *, fig=None, ax=None):
+    """
+    Animation of Qubism plot for pure states of many qudits.
+    Works best for spin chains, especially with even number of particles
+    of the same dimension.  Allows to see entanglement between first
+    2k particles and the rest.
+
+    .. note::
+
+        colorblind_safe does not apply because of its unique colormap
+
+    Parameters
+    ----------
+    ket : list of Qobj instances
+        Pure states for animation.
+
+    theme : 'light' or 'dark', default='light'
+        Set coloring theme for mapping complex values into colors.
+        See: complex_array_to_rgb.
+
+    how : 'pairs', 'pairs_skewed' or 'before_after', default='pairs'
+        Type of Qubism plotting.  Options:
+
+        - 'pairs' - typical coordinates,
+        - 'pairs_skewed' - for ferromagnetic/antriferromagnetic plots,
+        - 'before_after' - related to Schmidt plot (see also: plot_schmidt).
+
+    grid_iteration : int, default=1
+        Helper lines to be drawn on plot.
+        Show tiles for 2*grid_iteration particles vs all others.
+
+    legend_iteration : int or 'grid_iteration' or 'all', default=0
+        Show labels for first ``2*legend_iteration`` particles.  Option
+        'grid_iteration' sets the same number of particles as for
+        grid_iteration.  Option 'all' makes label for all particles.  Typically
+        it should be 0, 1, 2 or perhaps 3.
+
+    fig : a matplotlib figure instance, optional
+        The figure canvas on which the plot will be drawn.
+
+    ax : a matplotlib axis instance, optional
+        The axis context in which the plot will be drawn.
+
+    Returns
+    -------
+    fig, ani : tuple
+        A tuple of the matplotlib figure and the animation instance
+        used to produce the figure.
+
+    Notes
+    -----
+    See also [1]_.
+
+    References
+    ----------
+    .. [1] J. Rodriguez-Laguna, P. Migdal, M. Ibanez Berganza, M. Lewenstein
+       and G. Sierra, *Qubism: self-similar visualization of many-body
+       wavefunctions*, `New J. Phys. 14 053028
+       <https://dx.doi.org/10.1088/1367-2630/14/5/053028>`_, arXiv:1112.3560
+       (2012), open access.
+    """
+
+    fig, ani = plot_qubism(ket, theme, how, grid_iteration,
+                           legend_iteration, fig=fig, ax=ax)
+
+    plt.close(fig)
```

Comment:
Why do you close the figure?
It stop me from having it shown in the notebook with `%matplotlib notebook`.

Reply:
Thanks. This is a mistake.